mjml-ruby.gemspec

```diff
@@ -11,16 +11,16 @@ Gem::Specification.new do |s|
   s.homepage             = 'https://github.com/kolybasov/mjml-ruby'
   s.summary              = 'MJML parser and template engine for Ruby'
   s.license              = 'MIT'
-  s.post_install_message = 'Don\'t forget to run $ npm install -g mjml@^3.0'
 
   s.files                = `git ls-files app lib`.split("\n")
   s.test_files           = s.files.grep(%r{^(test|spec|features)/})
   s.platform             = Gem::Platform::RUBY
   s.require_paths        = ['lib']
 
-  s.add_runtime_dependency 'dry-configurable', '~> 0.1', '>= 0.1.3'
-
   s.add_development_dependency 'rake'
+  s.add_development_dependency 'railties'
+  s.add_development_dependency 'actionmailer'
+  s.add_development_dependency 'tzinfo-data'
```

Comment:
Could you add please versions constraints?

Reply:
@m-basov sure, up to you. I generally leave the dev deps unconstrained to avoid the noise or issues/prs to "bump versions". And since these are dev deps, the users don't care.

Reply:
I usually specify at least major version to avoid random breakage on CI.